package.json

```diff
@@ -755,11 +755,11 @@
                 },
                 {
                     "command": "aws.stepfunctions.createStateMachineFromTemplate",
-                    "when": "!isCloud9"
+                    "when": "false"
```

Comment:
should this be true? same question for the others...

Reply:
Yes. Changed to true.

Reply:
Actually, setting them to true does not work.  Removing them entirely works.  Tested it in C9 to be sure.